backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -186,6 +186,15 @@ type DType =
     | TFn _ -> true
     | _ -> false
 
+/// Used to constrain the type of a variable
+///
+/// e.g. `when 'T: comparison` in
```

Comment:
I really don't think we should go down this road (at the very least without a plan, but maybe not even then). I think simply naming the type parameters should be plenty.

Reply:
I agree these aren't DTypes though.

Reply:
Agreed, see https://github.com/darklang/dark/pull/4808#issuecomment-1480640133 - removing